core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -1,239 +1,242 @@
-registry.svc.ci.openshift.org/origin/4.7:telemeter quay.io/openshift/origin-telemeter:4.7 quay.io/openshift/origin-telemeter:4.7.0 quay.io/openshift/origin-telemeter:latest
-registry.svc.ci.openshift.org/origin/4.7:template-service-broker quay.io/openshift/origin-template-service-broker:4.7 quay.io/openshift/origin-template-service-broker:4.7.0 quay.io/openshift/origin-template-service-broker:latest
-registry.svc.ci.openshift.org/origin/4.7:template-service-broker-operator quay.io/openshift/origin-template-service-broker-operator:4.7 quay.io/openshift/origin-template-service-broker-operator:4.7.0 quay.io/openshift/origin-template-service-broker-operator:latest
-registry.svc.ci.openshift.org/origin/4.7:testcontent quay.io/openshift/origin-testcontent:4.7 quay.io/openshift/origin-testcontent:4.7.0 quay.io/openshift/origin-testcontent:latest
-registry.svc.ci.openshift.org/origin/4.7:tests quay.io/openshift/origin-tests:4.7 quay.io/openshift/origin-tests:4.7.0 quay.io/openshift/origin-tests:latest
-registry.svc.ci.openshift.org/origin/4.7:thanos quay.io/openshift/origin-thanos:4.7 quay.io/openshift/origin-thanos:4.7.0 quay.io/openshift/origin-thanos:latest
-registry.svc.ci.openshift.org/origin/4.7:tools quay.io/openshift/origin-tools:4.7 quay.io/openshift/origin-tools:4.7.0 quay.io/openshift/origin-tools:latest
-registry.svc.ci.openshift.org/origin/4.7:upi-installer quay.io/openshift/origin-upi-installer:4.7 quay.io/openshift/origin-upi-installer:4.7.0 quay.io/openshift/origin-upi-installer:latest
-registry.svc.ci.openshift.org/origin/4.7:vertical-pod-autoscaler quay.io/openshift/origin-vertical-pod-autoscaler:4.7 quay.io/openshift/origin-vertical-pod-autoscaler:4.7.0 quay.io/openshift/origin-vertical-pod-autoscaler:latest
-registry.svc.ci.openshift.org/origin/4.7:vertical-pod-autoscaler-operator quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.7 quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.7.0 quay.io/openshift/origin-vertical-pod-autoscaler-operator:latest
-registry.svc.ci.openshift.org/origin/4.7:windows-machine-config-bootstrapper-tests quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:4.7 quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:4.7.0 quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:latest
-registry.svc.ci.openshift.org/origin/4.7:windows-machine-config-operator-test quay.io/openshift/origin-windows-machine-config-operator-test:4.7 quay.io/openshift/origin-windows-machine-config-operator-test:4.7.0 quay.io/openshift/origin-windows-machine-config-operator-test:latest
-registry.svc.ci.openshift.org/ocp/4.7:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.7 quay.io/openshift/origin-operator-marketplace:4.7.0 quay.io/openshift/origin-operator-marketplace:latest
+registry.ci.openshift.org/origin/4.7:ansible quay.io/openshift/origin-ansible:4.7 quay.io/openshift/origin-ansible:4.7.0 quay.io/openshift/origin-ansible:latest
```

Comment:
What are the new entries here?  It might be better to split the regeneration from Make into its own PR and we can merge that first (that doesn't change to app.ci) and then we can do the api -> app move in this PR.  We're likely going to have to fix whatever added lines showed up (possibly by purging them from streams)

Reply:
There was one inappropriate image problem (marketplace) - I pinged evan to see what it is was and once we sort it out we can resolve this

Reply:
@smarterclayton 

DPTP has https://issues.redhat.com/browse/DPTP-1766 to implement for generating the mapping files.
Not sure how critical to keep the content up to date.
Or we could just move on with migration and update the files when the tool is ready.